content/en/docs/azure/azureEndtoEnd.md

```diff
@@ -58,7 +58,7 @@ To create an ML workspace:
 
 1. Go to [the Azure portal](https://portal.azure.com) and click on your resource group.
 2. Select the **add a new resource** option.
-3. Search for **Machine Learning Studio Workspace** and use the default options, taking note of the name you decide for it.
+3. Search for **Machine Learning** and use the default options, taking note of the name you decide for it.
```

Comment:
Recommendation: split this into two sentences. For example:

Search for **Machine Learning** and use the default options. Take note of the name you choose for this resource.